test/unix/common.mli

```diff
@@ -22,9 +22,10 @@ module Index : Index.Private.S with type key = Key.t and type value = Value.t
 module Make_context (Config : sig
   val root : string
 end) : sig
-  type t = {
+  type t = private {
```

Comment:
why are you using private here?

Reply:
It's a type of values that is only _consumed_ in external code, never constructed – by design rather than coincidence, since there are internal invariants / log messages unavailable to consumers. I put `private` to make that "destruct only" intended usage explicit.

Reply:
I see, thanks!